contracts/stellar-interchain-token-service/src/tests/migrate.rs

```diff
@@ -78,6 +74,15 @@ fn migrate_native_interchain_token_succeeds() {
         assert_ok!(InterchainTokenService::__migrate(&env, migration_data));
     });
 
+    env.as_contract(&its_client.address, || {
```

Comment:
avoid calling entrypoints using `env.as_contract` since it allows doing anything